text/0000-rustdoc-json.md

```diff
@@ -366,9 +368,9 @@ Name          | Type     | Description
 `is_unsafe`   | bool     | Whether this impl is for an unsafe trait.
 `generics`    | [Generics](#Generics) | Information about the impl's type parameters and `where` clauses.
 `provided_trait_methods` | [String] | The list of names for all provided methods in this impl block. This is provided for ease of access if you don't need more information from the `items` field.
-`trait`       | [Type](#Type) | The trait being implemented or `null` if the impl is "inherent".
+`trait`       | [Type](#Type) | (*Optional*) The trait being implemented or `null` if the impl is "inherent".
```

Comment:
Should we explain what inherent means?

```rust
pub struct S;
impl S { ... }
```

Reply:
yeah i definitely had to look that up when i read the code.